### PR TITLE
week1assesment-lc

### PR DESCRIPTION
**Pull request by LeoContreras93**
# Submitting Week 1 Assessment

### ✅ Completed Files
Select the following as confirmation that the files in this repository are complete.
- [x] Coding Challenges
- [x] Coding Concepts
- [x] Interview Questions

### 📝 Feedback
If you would like specific feedback on a particular problem please leave details here:
### I also struggled with understanding question #4 and why the built in method didn't necessarily work?

### CODE REVIEW

File: code-challenges.js [hunk 1]
```diff
@@ -16,41 +16,93 @@ const fruit2 = "banana"
...
+  console.log(stringCombo(fruit1, fruit2))
+  console.log(stringCombo(fruit3, fruit4))
+  //banana
+  //cherry
+
 
 // --------------------1) Create a function that takes in any given number and determines if the number is below boiling point, at boiling point, or above boiling point. Boiling point is 212 degrees Fahrenheit. Use the test variables provided below. Expected output: "42 is below boiling point", "350 is above boiling point", "212 is at boiling point"
 
-// Pseudo code:
+// Pseudo code: so any of the 3 test variables i place in console will return an answer about the tempature being below,at or above boiling point. tempature is my special veriable and I am invoking the test variables in the console
```
Comment (elysemontano) [inline code-challenges.js]: I would really like to see your pseudocode display the thought process of the logic you have written below.

File: code-challenges.js [hunk 2]
```diff
@@ -16,41 +16,93 @@ const fruit2 = "banana"
...
 const currentCohort = "Delta 2022"
+var cohort = currentCohort.split("")
+console.log(cohort.reverse().join("")) // 2202 atleD
+
 
 
 // --------------------4) Create a statement that will evaluate whether each number in an array is even or odd. Use the test variable provided below. Expected output: "odd" "even" "odd" "even" "odd" "even"
 
-// Pseudo code:
+// Pseudo code: unable to get the result of odd even, tried creating global variable with odd and even
```
Comment (elysemontano) [inline code-challenges.js]: I appreciate that you are mentioning what you are struggling with, but I need to see pseudocode that reflects the current logic shown below.  That being said, how are you accessing the current value in your array inside the for loop?  Also, double-check your statement on line 89.

File: code-challenges.js [hunk 3]
```diff
@@ -16,41 +16,93 @@ const fruit2 = "banana"
...
+
+
+     
+
 
 
 // --------------------5) Create a function that takes in a set of any two numbers and subtracts the smaller number from the larger number using the sets of test variables provided below. Expected output: 42, 3
 
-// Pseudo code:
+// Pseudo code: I am not sure how to set this up where i put in 58-100 to show 42 and vice versa
```
Comment (elysemontano) [inline code-challenges.js]: Same thing as I mentioned above, I need to see your pseudocode here, how did you break down the problem to get the result currently?

File: coding-concepts.js [hunk 4]
```diff
@@ -37,23 +37,23 @@ const languages = ["JavaScript", "Ruby", "Python", "C++"]
...
 
 // --------------------4) What will this log?
 
 const weekendDays = ["saturday", "sunday"]
 // console.log(weekendDays.toUpperCase())
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: my expectation was that it would return all letters capatizile. 
+// b) Verify and explain: an error occur with the built in method stating it is not a function, the other reason i think it wont work also is the strings are built into an array??
```
Comment (elysemontano) [inline coding-concepts.js:50]: .toUpperCase() is a built-in method that only works on strings

File: interview-questions.md [hunk 5]
```diff
@@ -7,62 +7,67 @@ Then, research the question to expand on your answer. Even if you feel you have
...
 
 
 
 1. What is git? What is the difference between git and Github?
 
-  Your answer:
+  Your answer: git is version control used on your local machine to send or receive information from the cloud(github). github is a website which contains repositiories where one can store their code for everyone to see.
 
-  Researched answer:
+  Researched answer: Simply put, Git is a version control system that lets you manage and keep track of your source code history. GitHub is a cloud-based hosting service that lets you manage Git repositories
```
Comment (elysemontano) [inline interview-questions.md]: Researched answers must be formed **in your own words**.

File: interview-questions.md [hunk 6]
```diff
@@ -7,62 +7,67 @@ Then, research the question to expand on your answer. Even if you feel you have
...
 
 
 
 2. Which JavaScript operators will return a Boolean value?
 
-  Your answer:
+  Your answer: Equality Operators, there are three types, loose equality, strict equality and logical operator
```
Comment (elysemontano) [inline interview-questions.md]: One-line answers are meeting the bare minimum requirement for this section and will come off as uninterested to an employer in an interview.  I would highly recommend expanding and discussing what each one does.

**Review by elysemontano — CHANGES_REQUESTED** [review]
Changes requested: All researched answers must be formed **in your own words**. No copying and pasting from any resources. I want to know your understanding of the concept, not googles. I will approve when your interview answers are updated.  Also, please make sure to add valid pseudocode for problems 1, 4, and 5 in code challenges.

**Comment by LeoContreras93** [comment]
changes made to week 1 assesment

**Review by elysemontano — APPROVED** [review]
Great job on the updates!  Approved!

**Pull request merged**